### ISSUE TITLE
Failed to load error message

### ISSUE THREAD

**Issue by robinlovelace-ate** [acteng/atip#206]
As shown below.

![image](https://github.com/acteng/atip/assets/122299965/3d5f4e7c-0f06-46eb-a235-a00fe22316d4)

Browser logs...

```
GJ changed, saving to local storage
worker-8a25f921.js:6 Grabbing A/B Street map model data from https://atip.uk/route-info-dev/West Yorkshire Combined Authority.bin.gz
nested-5e5e6e65.js:59 Grabbing https://atip.uk/route-snappers-dev/West Yorkshire Combined Authority.bin.gz
atip.uk/route-snappers-dev/West%20Yorkshire%20Combined%20Authority.bin.gz:1     Failed to load resource: net::ERR_CONNECTION_RESET
nested-5e5e6e65.js:59 Route tool broke: TypeError: network error
nested-5e5e6e65.js:64 Uncaught (in promise) TypeError: Failed to fetch
nested-5e5e6e65.js:63 Stopping old mode edit-attribute
nested-5e5e6e65.js:63 Starting new mode route
nested-5e5e6e65.js:59 Uncaught TypeError: Cannot read properties of undefined (reading 'isActive')
    at Zu.u (nested-5e5e6e65.js:59:6807)
    at h (nested-5e5e6e65.js:63:8062)
    at HTMLButtonElement.l (nested-5e5e6e65.js:45:11160)
```

**Comment by robinlovelace-ate** [comment]
Source: https://acteng.github.io/atip/scheme.html?authority=West%20Yorkshire%20Combined%20Authority#12.05/53.81561/-1.54313

**Comment by robinlovelace-ate** [comment]
Can reproduce on Edge (previous test on Chrome).

**Comment by dabreegster** [comment]
Working fine for me right now. `ERR_CONNECTION_RESET` means your internet might've been flaky, or possibly Cloudfront (but that seems less likely).

**Comment by robinlovelace-ate** [comment]
Working today. Closing in favour of #207 which when fixed will mitigate any issues.

**Comment by robinlovelace-ate** [comment]
As tested with @Pete-Y-CS it's happening again on my laptop

<img width="683" alt="image" src="https://github.com/acteng/atip/assets/122299965/9ef8d4b2-28aa-48f5-a0ad-0efe4c104a5f">

**Comment by robinlovelace-ate** [comment]
Confirmed, this endpoint is blocked on my laptop. No idea why this is happening on intermittently, only seems to happen when I'm at home on work laptop. https://atip.uk/route-snappers/v2/Leeds.bin.gz

can anyone else reproduce @Pete-Y-CS and @Sparrow0hawk ?

**Comment by robinlovelace-ate** [comment]
Thought: would "Load local binary network file" be an option? Could be linked with fact it's a .gz file?

**Comment by Sparrow0hawk** [comment]
I imagine this is a Zscaler issue with DfT laptops working from home blocking the download of a file `.gz` file (it'll probably hate `.bin` files too)

**Comment by Pete-Y-CS** [comment]
> Confirmed, this endpoint is blocked on my laptop. No idea why this is happening on intermittently, only seems to happen when I'm at home on work laptop. https://atip.uk/route-snappers/v2/Leeds.bin.gz
> 
> can anyone else reproduce @Pete-Y-CS and @Sparrow0hawk ?

ahh sorry, yeah I had the same issue
